07-routing-gossip.md

```diff
@@ -289,6 +289,12 @@ The receiving node MAY use `rgb_color` and `alias` to reference nodes in interfa
 
 ### Rationale
 
+The `timestamp` field is used by nodes for pruning (either if it's too
+far in the future, or if it's been two weeks with no update), so it
+makes sense to have it be a UNIX timestamp (ie. seconds since UTC
+1970-01-01).  It can't be a hard requirement, however, given the possible case
```

Comment:
Shouldn't this last sentence be in the `channel_update` section? Also, it is not clear that the first requirement applies to both `node_announcement` and `channel_update`.

Maybe either duplicate/reference the requirement, or move this whole paragraph to the "Pruning the Network View" section?

Reply:
Oops, yes, I edited the wrong section!  I fixed it up...  Good catch!